code/modules/mob/living/carbon/give.dm

```diff
@@ -0,0 +1,57 @@
+/mob/living/carbon/verb/give()
+	set category = "IC"
+	set name = "Give"
+	set src in view(1)
+	if(src.stat == CONSCIOUS || usr.stat == CONSCIOUS || src.client == null)
```

Comment:
This should be ```if(stat != CONSCIOUS)``` same goes for the ```usr.stat``` too.
because what this says now is that if you're awake, it won't work.